pytorch_lightning/trainer/trainer.py

```diff
@@ -1920,9 +1919,9 @@ def slurm_job_id(self) -> Optional[int]:
         return SLURMEnvironment.job_id()
 
     @property
-    def lightning_optimizers(self) -> List[LightningOptimizer]:
+    def lightning_optimizers(self) -> Dict[int, LightningOptimizer]:
```

Comment:
was this a bug before that the typehint was wrong?

Reply:
Yes

Reply:
could you add this to the changelog under a fix?

Reply:
sure I will add it. But does this really qualify as a bug? It is just a typehint

Reply:
sure I will add it. I didn't really consider a wrong typehint a bug